.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: ci-workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository info'
+        required: true
+      branch:
+        description: 'Name of the branch'
+        required: true
+      target_repository:
+        description: 'Repository name of the terratest'
```

Comment:
Repository name of the terratest repo, perhaps?

Reply:
🤦‍♂️ you're right, fixed..